sklearn/impute/_base.py

```diff
@@ -225,6 +225,11 @@ class SimpleImputer(_BaseImputer):
 
         .. versionadded:: 1.2
 
+        .. versionchanged:: 1.6
+        Currently, when `keep_empty_feature=False` and `strategy="constant"`,
+        empty features are not dropped. This behaviour will change in version
+        1.8. Set `keep_empty_feature=True` to preserve this behaviour.
```

Comment:
This should solve the documentation failure, `sphinx` is quite stubborn about those :).

```suggestion
        .. versionchanged:: 1.6
           Currently, when `keep_empty_feature=False` and `strategy="constant"`,
           empty features are not dropped. This behaviour will change in version
           1.8. Set `keep_empty_feature=True` to preserve this behaviour.
```